wsd/DocumentBroker.cpp

```diff
@@ -1400,8 +1400,7 @@ DocumentBroker::updateSessionWithWopiInfo(const std::shared_ptr<ClientSession>&
 
     bool disablePresentation = wopiFileInfo->getDisableExport() || wopiFileInfo->getHideExportOption();
     // the new slideshow supports watermarking, anyway it's still an experimental features
-    // TODO remove the following line when the new slideshow is no more experimental
-    disablePresentation = disablePresentation || (!ConfigUtil::getBool("experimental_features", false) && !watermarkText.empty());
+    disablePresentation = disablePresentation || ConfigUtil::getBool("slideshow", false);
```

Comment:
also logic is not preserved here:
We want to disable presentation when new slideshow is OFF and watermark text is SET (because svg presentation doesn't support watermarking but canvas one yes)
so it shoudl be `... || ( !canvasSlideshowEnabled && !watermarkText.empty() )`